src/main/resources/GoogleApps/GoogleAppsConfigSheet.vm

```diff
@@ -13,82 +13,95 @@ $xwiki.ssx.use('GoogleApps.GoogleAppsConfigClass')
     $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}")##
 #end
 #macro(tp $n $p)##
-    $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p)##
+    #set($prf="GoogleApps.GoogleAppsConfigClass_")
+    #if($n.startsWith("googleapps"))#set($prf="")#end
```

Comment:
Is there any reason to not have a proper indentation in this macro?

Reply:
You probably meant the if. I tend to like compact such statements but I changed it nonetheless to three lines.